internal/ui/format.go

```diff
@@ -8,13 +8,13 @@ import (
 func FormatBytes(c uint64) string {
 	b := float64(c)
 	switch {
-	case c > 1<<40:
-		return fmt.Sprintf("%.3f TiB", b/(1<<40))
```

Comment:
Is there a reason to use `%3.3f` here but nowhere else?

Reply:
That's fine. The 3.3f was left over from an experiment that I forgot to undo.